docs/overlay-hld.adoc

```diff
@@ -767,39 +769,44 @@ the CSV in which the grouping tool will output its groupings.
 3. Linker autogrouping - the linker will put any overlay symbol that has
 not been assigned to a group into its own group.
 
+Linker autogrouping is the default. Grouping file and grouping tool are optional
+and using both is an error.
+
+===== Tables
+
+The linker will construct the overlay offset table and multi-group table (if
+multi-groups are present).
+
 ===== Sections
 
 The linker will populate the following output sections:
 
-.ovlgrps - contains each overlay group. Referred to as ``overlay area` in the
+* `.ovlgrps` - contains each overlay group. Referred to as ``overlay area` in the
 requirements document.
-* The first group contains the overlay offset table followed by the multi-group
-table (if multi-groups are present).
-* Each input section `.ovlinput.*` for each overlay symbol is copied into the
-group(s) to which it has been assigned.
-
-.ovlcache - the overlay heap into which overlay groups are loaded at runtime by
-the RT-Engine. Referred to as ``heap area'' in the requirements document.
+  * The first group contains the overlay offset table followed by the multi-group
+  table (if multi-groups are present).
+  * Each input section `.ovlinput.*` for each overlay symbol is copied into the
```

Comment:
```  * Each input section `.ovlinput.*` for each overlay symbol is copied into the
  group(s) to which it has been assigned.```
Not sure I understand the intention

Reply:
Is it clearer if I change that text to `Each overlay function and data is copied to the group(s) to which it has been assigned.`?

Reply:
Yes, I think it will be clearer

Reply:
Thanks, now updated.